LOAN/Loans/LoanBasicTerms.rdf

```diff
@@ -143,14 +143,14 @@
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-agr-ctr;hasContractThirdParty"/>
-				<owl:someValuesFrom rdf:resource="&fibo-fnd-agr-ctr;ContractThirdParty"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-agr-ctr;hasContractualElement"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-dbt;DebtTerms"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-agr-ctr;hasContractualElement"/>
-				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-dbt;DebtTerms"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-agr-ctr;hasThirdParty"/>
```

Comment:
This change should not be in this PR - it's nothing to do with addresses.

Reply:
Agreed - I only fixed it because hygiene testing highlighted it.  I'll ignore hygiene testing going forward if you prefer.